include/pick_ik/robot.hpp

```diff
@@ -27,8 +32,8 @@ struct Robot {
                      moveit::core::JointModelGroup const* jmg,
                      std::vector<size_t> tip_link_indices) -> Robot;
 
-    /** @brief Returns a random valid configuration. */
-    auto get_random_valid_configuration() const -> std::vector<double>;
+    /** @brief Sets a variable vector to a random valid configuration. */
```

Comment:
Can you be specific with the definition of "valid"? Random joint angles that are within the defined limits?